content/applications/productivity/calendar/outlook.rst

```diff
@@ -41,37 +69,71 @@ URI (URL) followed by `/microsoft_account/authentication`.
 For more information on the restrictions and limitations of URIs, `check this page <https://docs.
 microsoft.com/en-us/azure/active-directory/develop/reply-url>`_.
 
-Regarding the application credentials, the user *must* add a client secret, which allows Odoo to
-authenticate itself, requiring no interaction from the user's side. :guilabel:`Certificates` are
-optional.
+Finally, click :guilabel:`Register` to complete the application registration. The
+:guilabel:`Application (client) ID` is produced. Copy this value as it is needed later, in the
+:ref:`outlook_calendar/odoo_setup`.
+
+.. image:: outlook/app-client-id.png
+   :align: center
+   :alt: Application client ID highlighted in the essentials section of the newly created
+         application.
+
+Create client secret
+--------------------
 
-To do add a client secret, click :guilabel:`Add a certificate or secret` and then click
-:guilabel:`New client secret`. Next, type a :guilabel:`Description` and select when the client
-secret :guilabel:`Expires`.
+The second credential needed to complete the synchronization of the Microsoft Outlook Calendar is
+the *Client Secret*. The user **must** add a client secret, as this allows Odoo to authenticate
+itself, requiring no interaction from the user's side. :guilabel:`Certificates` are optional.
```

Comment:
At this point in your instructions, is the word 'Certificates' appearing on the UI, if the reader were to follow this doc move-by-move? If so, then absolutely keep the GUI for 'Certificates' -- it not, I'd suggest applying some italics here, instead.